backend/app/dto/app.go

```diff
@@ -92,6 +92,7 @@ type AppProperty struct {
 	Website            string   `json:"website"`
 	Github             string   `json:"github"`
 	Document           string   `json:"document"`
+	Version            float64  `json:"version"`
 }
 
 type AppConfigVersion struct {
```

Comment:
This seems to be an incomplete piece of C# code related to setting up a version control system (GitHub) with some properties like website, GitHub URL, Document file location, and current document update version which is represented as float64. However, it lacks necessary spaces around operators (`+`, `-`) and semicolons to maintain consistent formatting.

For better quality and more readable code structure:

```csharp
public partial class AppProperty {

  public string Website { get; set; }

  public string Github { get; set; }
  
}

public enum AppConfigVersion {}

```

As you can see, each property and enumeration value are on their own lines so that spacing is standardized throughout the code block.